packs/_source/monsters/dragon/adult-green-dragon.json

```
@@ -2297,7 +2297,7 @@
           "base": {
             "number": 2,
             "denomination": 6,
-            "bonus": "",
+            "bonus": 6,
```

No need to change this (in fact, this damage data isn't actually used, base weapon damage isn't added to saves, only attacks).

```suggestion
            "bonus": "",
```